modules/monitoring-exploring-the-visualized-metrics.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="exploring-the-visualized-metrics_{context}"]
+= Exploring the visualized metrics
+
+After running the queries, the metrics are displayed on the interactive plot. The X axis of the plot represents time. The Y axis represents the metrics values. Each metric is shown as a colored graph. You can manipulate the plot and explore the metrics.
+
+.Procedure
+
+. Initially, all metrics from all enabled queries are shown on the plot. You can select which metrics are shown.
+* To hide all metrics from a query, click the action button of the query and click "Hide all series".
+* To hide a specific metric, go to the query table and click the colored square near the metric name.
+. To zoom into the plot and change the shown time range, do one of the following:
++
+--
+* Visually select the time range by clicking and dragging on the plot horizontally.
+* Use the menu in the left upper corner to select the time range.
+--
++
+To reset the time range, click "Reset Zoom".
+. To see outputs of all queries at a specific point in time, hold the mouse cursor on the plot at that point. Query outputs will appear in a popup box.
```

Comment:
I wonder about _box_ vs _window_, but I don't think we have any codified style for referring to the UI elements in the Web UI yet.

Reply:
Calling that small popup rectange a window would be confusing. It think this will be commonly understood much better as a "popup box".